packages/frontend/src/components/account/EthAccountDialog.tsx

```diff
@@ -47,11 +47,7 @@ const EthAccountDialog = ({
       initialFocus={contentRef}
     >
       <div className="flex items-center justify-center min-h-screen">
-        <Dialog.Overlay className="fixed inset-0 bg-gray-800 opacity-30" />
-        <div
-          className="relative max-w-xl p-6 mx-auto bg-white rounded dark:bg-slate-900"
-          ref={contentRef}
-        >
+        <div className="p-10 praise-box-defaults" ref={contentRef}>
```

Comment:
Personally, I like having a background overlay that "mutes" the page content when a dialog is opened. I feel like it improves the distinction between the dialog and the background content. Alternatively maybe we could increase the dialog's shadow area? 

This screenshot shows how the dialog is hard to distinguish from the content behind it:
![image](https://user-images.githubusercontent.com/159270/174448539-b23e0ff5-8f0d-419d-bdad-3132b8b36a4c.png)

Reply:
Yeah, we used to have that, right? I agree. I'll bring it back, I must have accidentally destroyed something, just adding back the deleted styles don't bring back the background. Good catch, I forgot this one.

Reply:
Fixed.